test/helpers/deploys_helper_test.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 require_relative '../test_helper'
 
-SingleCov.covered! uncovered: 1
+SingleCov.covered! uncovered: 11
```

Comment:
ouch

Reply:
yeah! this is actually not true, but given that I am overriding the method with the new deploys helper, singlecov thinks these lines are not covered anymore, but they actually are... nevertheless, I hope I can revert this change if I take the decorator + hook path instead of overriding the method directly.